python/tests/server/test_http.py

```diff
@@ -3,6 +3,7 @@
 
 from PIL import Image
 import pytest
+import unittest.mock as mock
```

Comment:
This is a slightly unusual way of saying `from unittest import mock` but not a big deal in test code :)